tests/linalg_test.py

```diff
@@ -1480,6 +1480,32 @@ def testRsf2csf(self, shape, dtype):
                               args_maker, tol=tol)
       self._CompileAndCheck(jsp.linalg.rsf2csf, args_maker)
 
+  @parameterized.named_parameters(
+        jtu.cases_from_list({
+            "testcase_name":
+            "_shape={}_disp={}".format(jtu.format_shape_dtype_string(shape, dtype), disp),
+            "shape": shape, "dtype": dtype, "disp": disp
+        } for shape in [(1, 1), (5, 5), (20, 20), (50, 50)]
+            for dtype in float_types + complex_types
+              for disp in [True, False]))
+  @jtu.skip_on_devices("gpu", "tpu")
+  def testFunm(self, shape, dtype, disp):
+      def func(x):
+        return x**-2.718
+      rng = jtu.rand_default(self.rng())
+      args_maker = lambda: [rng(shape, dtype)]
+      if disp:
+        jnp_fun = lambda arr: jsp.linalg.funm(arr, func, disp=disp)
+        scp_fun = lambda arr: osp.linalg.funm(arr, func, disp=disp)
+      else:
+        # if disp is False then funm returns a tuple (array, error) and
+        # the error is ignored here given it is likely to differ
```

Comment:
We should not leave the error untested, particularly since it's not a straightforward computation. Why is it likely to differ? Can we test the value with an appropriate tolerance to make sure we're not returning entirely the wrong thing?

Reply:
In my initial experimentatio, I thought I noticed a large discrepancy. I've checked again and the discrepancy is less than I feared - I'll make sure the test here checks the error as well as the array.